src/ctapipe/image/muon/fitting.py

```diff
@@ -50,22 +65,75 @@ def kundu_chaudhuri_circle_fit(x, y, weights):
         np.sum(weights * ((center_x - x) ** 2 + (center_y - y) ** 2)) / weights_sum
     )
 
-    return radius, center_x, center_y
+    if dummy_errors_flag:
```

Comment:
These are magic numbers that don't seem to have any meaning. Where do they come from? Why do we need "dummy errors"?

Why not just fill "nan" in case you do not compute errors?